.releaserc.js

```diff
@@ -11,30 +12,41 @@ module.exports = {
       {
         assets: [
           {
-            path: ".artifacts/zitadel-linux-amd64/zitadel-linux-amd64.tar",
-            label: "zitadel-linux-amd64.tar",
+            path: ".artifacts/zitadel-linux-amd64/zitadel-linux-amd64.tar.gz",
+            label: "zitadel-linux-amd64.tar.gz",
           },
           {
-            path: ".artifacts/zitadel-linux-arm64/zitadel-linux-arm64.tar",
-            label: "zitadel-linux-arm64.tar",
+            path: ".artifacts/zitadel-linux-arm64/zitadel-linux-arm64.tar.gz",
+            label: "zitadel-linux-arm64.tar.gz",
           },
           {
-            path: ".artifacts/zitadel-windows-amd64/zitadel-windows-amd64.tar",
-            label: "zitadel-windows-amd64.tar",
+            path: ".artifacts/zitadel-windows-amd64/zitadel-windows-amd64.tar.gz",
+            label: "zitadel-windows-amd64.tar.gz",
           },
           {
-            path: ".artifacts/zitadel-windows-arm64/zitadel-windows-arm64.tar",
-            label: "zitadel-windows-arm64.tar",
+            path: ".artifacts/zitadel-windows-arm64/zitadel-windows-arm64.tar.gz",
+            label: "zitadel-windows-arm64.tar.gz",
           },
           {
-            path: ".artifacts/zitadel-darwin-amd64/zitadel-darwin-amd64.tar",
-            label: "zitadel-darwin-amd64.tar",
+            path: ".artifacts/zitadel-darwin-amd64/zitadel-darwin-amd64.tar.gz",
+            label: "zitadel-darwin-amd64.tar.gz",
           },
           {
-            path: ".artifacts/zitadel-darwin-arm64/zitadel-darwin-arm64.tar",
-            label: "zitadel-darwin-arm64.tar",
+            path: ".artifacts/zitadel-darwin-arm64/zitadel-darwin-arm64.tar.gz",
+            label: "zitadel-darwin-arm64.tar.gz",
           },
+          {
+            path: ".artifacts/checksums.txt",
+            label: "checksums.txt",
+          }
         ],
+        draftRelease: true,
+        successComment: false,
+        failComment: false,
+        labels: false,
+        releasedLabels: false,
+        addReleases: false,
+        failTitle: false,
```

Comment:
```suggestion
```

Reply:
only for testing the .tar.gz change, will be removed shortly